pyfr/__main__.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import sys
+sys.path.append('/home/gonzalo/PyFR_fork/PyFR/')
```

Comment:
So the path added in this line of code is the home directory to the PyFR?

Reply:
That was a mistake on my end, I'll fix it. It's just a small band aid to help me debug stuff.